### PR TITLE
Adding HDP discovery tool

### PR DESCRIPTION
**Pull request by aravindrox**
Adding discovery tool for HDP under upgrade-toolkit

### CODE REVIEW

File: upgrade-toolkit/HDP Discovery Tool/.gitignore [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Byte-compiled / optimized / DLL files
```
Comment (Chaffelson) [inline upgrade-toolkit/HDP Discovery Tool/.gitignore]: This should not be included

File: upgrade-toolkit/HDP Discovery Tool/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (Chaffelson) [inline upgrade-toolkit/HDP Discovery Tool/LICENSE]: This should not be included

**Review by Chaffelson — CHANGES_REQUESTED** [review]

**Review by Chaffelson — CHANGES_REQUESTED** [review]
Please can you remove spaces from paths

**Comment by aravindrox** [comment]
Removed the spaces in root directory, Removed LICENCE and .gitignore

**Pull request closed without merging**